SUASSystem/SUASSystem/sda_converter.py

```diff
@@ -61,11 +60,12 @@ def set_waypoint(self, new_waypoint):
         :type new_waypoint: Location
         """
         converted_waypoint = convert_to_point(self.initial_coordinates, new_waypoint)
-
+        print('waypoint')
+        print(converted_waypoint)
         self.obstacle_map.reset_waypoints()
         self.obstacle_map.add_waypoint(converted_waypoint)
 
-    def add_obstacle(self, obstacle_location, obstacle):
+    def add_obstacle(self, obstacle):
         """
         Add an obstacle to the obstacle map
```

Comment:
In function docs, description and items (`:param obstacles: ...`, `:type obstacle: ...`, `:return: ...`) should end with periods for readability